# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/generic/crashtests/crashtests.list
# Commit: f770099a60cb
# Full Hash: f770099a60cb34e14a40ee8170807972398aa219
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-06-29 09:58:05
# Description:
#   Bug 1562105 - Bail out if the style frame for the scrollable frame is null in ScrollFrameHelper::IsSmoothScroll. r=botond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D36303
# ==============================================================================

diff -r b8d39cd984a2 -r f770099a60cb layout/generic/crashtests/crashtests.list
--- a/layout/generic/crashtests/crashtests.list	Fri Jun 28 15:58:36 2019 +0000
+++ b/layout/generic/crashtests/crashtests.list	Fri Jun 28 16:45:54 2019 +0000
@@ -734,3 +734,4 @@
 pref(layout.css.grid-template-subgrid-value.enabled,true) load 1553824.html
 load 1346454-1.html
 load 1346454-2.html
+load 1562105.html